__tests__/index.tsx

```diff
@@ -151,8 +152,17 @@ describe('VideoRenderer', () => {
   });
 
   describe('actions', () => {
-    xit('should set video current time to passed time when navigate is called', () => {
 
+    let videoActions: VideoActions;
+    const children: RenderCallback = (videoElement: ReactNode, state: VideoState, actions: VideoActions) => {
+      videoActions = actions;
+      return videoElement;
+    };
+
+    it('should set video current time to passed time when navigate is called', () => {
+      const {component} = setup({children});
+      videoActions.navigate(10);
+      expect(component.state().currentTime).toEqual(10);
     });
 
     xit('should play the video when play action is called', () => {
```

Comment:
I think we can also add tests for this one without having to check the video element, just by checking `state.status`. What do you think?

Reply:
The problem is that the new status is assigned when the `onPlay` callback gets called by the video element when the `play` command is called, and the whole event wire-up is not implemented by `jsdom.